ui/redux/actions/comments.js

```diff
@@ -309,6 +309,17 @@ export function doCommentCreate(
           toastMessage = __('Unable to comment. This channel has blocked you.');
         }
 
+        if (error) {
+          const BLOCKED_WORDS_ERR_MSG = 'the comment contents are blocked by';
+
+          if (error.message === 'channel is blocked by publisher') {
```

Comment:
Now that `comment.Create` is going directly though commentron, can we access error codes instead of looking at the error message string?

Reply:
The code is the same for all errors `32000`, so I parsed the message instead.
```
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32000,
    "message": "words to block ? must be greater than length 3",
    "data": null
  },
  "id": 1
}
```
```
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32000,
    "message": "channel is blocked by publisher",
    "data": null
  },
  "id": 1
}
```
@tiger5226, is it possible to have unique codes?

Reply:
did you get an answer about error codes?

Reply:
Haven't. No reply yet.

Reply:
https://github.com/lbryio/lbry.go/issues/87 added an issue to the lbry.go package. The current errors package we use does not support error codes.